domains/logging-server.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "UntoldGam",
+        "email": "charlesobird+github@outlook.com"
+    },
+    "record": {
+        "A": 130.162.166.23
```

Comment:
IP needs to be in a array

Reply:
so quick

Reply:
I didn't mean for that file to be sent, the pr was done via the website. Is it alright to close this and make a new pr?

Reply:
I don't need that file, it's the charlesbird.json one that i need

Reply:
> I don't need that file, it's the charlesbird.json one that i need

delete it then